examples/model_compression/distill_lstm/args.py

```diff
@@ -98,12 +98,12 @@ def parse_args():
     )
 
     parser.add_argument(
-        "--teacher_path", type=str, help="Teacher model's path.")
+        "--teacher_dir", type=str, help="Teacher model's directory.")
 
     parser.add_argument(
         "--vocab_path",
         type=str,
-        default='/root/.paddlenlp/models/bert-base-uncased/bert-base-uncased-vocab.txt',
+        default='~/.paddlenlp/models/bert-base-uncased/bert-base-uncased-vocab.txt',
```

Comment:
python script may not recognize `'~/'`? `DATA_HOME + 'path/to/vocab'` may work.

Reply:
Thanks! Get~